cpu_fetcher.py

```diff
@@ -20,35 +20,52 @@ def _get_tweeted_product_types(self) -> List[str]:
         Return product types you want to appear in tweets (one for each)
         To tweet about all product types: self.database.find_distinct_product_types()
         """
-        return ["RYZEN 5 3600X", "RYZEN 9 3950X"]
+        return ["RYZEN 5 3600X", "RYZEN 9 3950X", "CORE I9 9900KF"]
 
     def _get_source_product_urls(self) -> Dict[type(Source), Dict[str, str]]:
         return {
             TopAchat: {
+                "Intel Core": "https://www.topachat.com/pages/produits_cat_est_micro_puis_rubrique_est_wpr_puis_ordre_est_P_puis_sens_est_ASC_puis_f_est_3-61|557-8827.html",
                 "AMD Ryzen": "https://www.topachat.com/pages/produits_cat_est_micro_puis_rubrique_est_wpr_puis_ordre_est_P_puis_sens_est_ASC_puis_f_est_3-63|557-8865,8660,8617,10578.html",
             },
             GrosBill: {
+                "Intel Core": "https://www.grosbill.com/3-processeur_intel-processeur_intel_core_i9-type-informatique#filtre_mini=-1&filtre_maxi=-1&filtre_marque=INTEL",
```

Comment:
https://www.grosbill.com/2-processeur_intel-cat-informatique?page=1&tri=w&filtre_page=100&mode=listing&filtre_type_produit=processeur is better